app/Resources/views/search/committees.html.twig

```diff
@@ -0,0 +1,15 @@
+{% extends 'search/layout.html.twig' %}
+
+{% block results %}
+    <a style="float: left; width: 200px; background-color: lightgrey; margin: 10px; display: block" href="#">
+        <h3>This is a committee name</h3>
+        <h4>
+            <small>
+              {{ '1 participant|%count% participants'|transchoice(5) }}
+              {% if 1 %}
```

Comment:
It' a fixture, to prepare the condition, but if you want I can remove this skeleton.